aws/Dockerfile

```diff
@@ -2,4 +2,4 @@ ARG REPOSITORY_BASE_PATH
 
 FROM ${REPOSITORY_BASE_PATH}:latest
 
-RUN pip install boto3 botocore
+RUN pip install boto3 --break-system-packages
```

Comment:
boto3 already consists of botocore (transitive dependency), so it makes zero sense to specify it

Reply:
Good catch 👍🏻